ill-fated-ducks/Bangazon-Site
# Jd product status

## Status
Choose:
**READY**


## Description
Allows a signed in user to view their listed products for sale, along with the current quantity and numbers sold.

## Todos

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
1. git fetch
2. git checkout jd-productStatus
3. dotnet run
4. Sign in as a user and click "My Products" in the navbar
5. You should see any products they have, with current inventory and quantity sold if applicable

## Impacted Areas in Application
Views/Products/MyProducts.cshtml
BangazonSite/Data/ApplicationDbContext.cs
BangazonSite/Data/SeedData.cs
BangazonSite/Models/ProductViewModels/MyProductsOrderVM.cs


## Link to Feature Ticket
#15
@ill-fated-ducks
